.circleci/config.yml

```diff
@@ -59,9 +59,9 @@ jobs:
-            docker build -t ld4p/sinopia_editor:dev --build-arg SPOOF_SINOPIA_SERVER=$SPOOF_SINOPIA_SERVER_DEV --build-arg TRELLIS_BASE_URL=$TRELLIS_BASE_URL_DEV --build-arg DEFAULT_PROFILE_SCHEMA_VERSION=$DEFAULT_PROFILE_SCHEMA_VERSION_DEV --build-arg SINOPIA_GROUP=$SINOPIA_GROUP_DEV --build-arg SINOPIA_URI=$SINOPIA_URI_DEV --build-arg AWS_COGNITO_DOMAIN=$AWS_COGNITO_DOMAIN_DEV --build-arg COGNITO_CLIENT_ID=$COGNITO_CLIENT_ID_DEV --build-arg COGNITO_USER_POOL_ID=$COGNITO_USER_POOL_ID_DEV .
-            docker build -t ld4p/sinopia_editor:stage --build-arg SPOOF_SINOPIA_SERVER=$SPOOF_SINOPIA_SERVER_STAGE --build-arg TRELLIS_BASE_URL=$TRELLIS_BASE_URL_STAGE --build-arg DEFAULT_PROFILE_SCHEMA_VERSION=$DEFAULT_PROFILE_SCHEMA_VERSION_STAGE --build-arg SINOPIA_GROUP=$SINOPIA_GROUP_STAGE --build-arg SINOPIA_URI=$SINOPIA_URI_STAGE --build-arg AWS_COGNITO_DOMAIN=$AWS_COGNITO_DOMAIN_STAGE --build-arg COGNITO_CLIENT_ID=$COGNITO_CLIENT_ID_STAGE --build-arg COGNITO_USER_POOL_ID=$COGNITO_USER_POOL_ID_STAGE .
-            docker build -t ld4p/sinopia_editor:prod --build-arg SPOOF_SINOPIA_SERVER=$SPOOF_SINOPIA_SERVER_PROD --build-arg TRELLIS_BASE_URL=$TRELLIS_BASE_URL_PROD --build-arg DEFAULT_PROFILE_SCHEMA_VERSION=$DEFAULT_PROFILE_SCHEMA_VERSION_PROD --build-arg SINOPIA_GROUP=$SINOPIA_GROUP_PROD --build-arg SINOPIA_URI=$SINOPIA_URI_PROD --build-arg AWS_COGNITO_DOMAIN=$AWS_COGNITO_DOMAIN_PROD --build-arg COGNITO_CLIENT_ID=$COGNITO_CLIENT_ID_PROD --build-arg COGNITO_USER_POOL_ID=$COGNITO_USER_POOL_ID_PROD .
+            docker build -t ld4p/sinopia_editor:dev --build-arg USE_FIXTURES=$USE_FIXTURES_DEV --build-arg TRELLIS_BASE_URL=$TRELLIS_BASE_URL_DEV --build-arg DEFAULT_PROFILE_SCHEMA_VERSION=$DEFAULT_PROFILE_SCHEMA_VERSION_DEV --build-arg SINOPIA_GROUP=$SINOPIA_GROUP_DEV --build-arg SINOPIA_URI=$SINOPIA_URI_DEV --build-arg AWS_COGNITO_DOMAIN=$AWS_COGNITO_DOMAIN_DEV --build-arg COGNITO_CLIENT_ID=$COGNITO_CLIENT_ID_DEV --build-arg COGNITO_USER_POOL_ID=$COGNITO_USER_POOL_ID_DEV .
+            docker build -t ld4p/sinopia_editor:stage --build-arg USE_FIXTURES=$USE_FIXTURES_STAGE --build-arg TRELLIS_BASE_URL=$TRELLIS_BASE_URL_STAGE --build-arg DEFAULT_PROFILE_SCHEMA_VERSION=$DEFAULT_PROFILE_SCHEMA_VERSION_STAGE --build-arg SINOPIA_GROUP=$SINOPIA_GROUP_STAGE --build-arg SINOPIA_URI=$SINOPIA_URI_STAGE --build-arg AWS_COGNITO_DOMAIN=$AWS_COGNITO_DOMAIN_STAGE --build-arg COGNITO_CLIENT_ID=$COGNITO_CLIENT_ID_STAGE --build-arg COGNITO_USER_POOL_ID=$COGNITO_USER_POOL_ID_STAGE .
+            docker build -t ld4p/sinopia_editor:prod --build-arg USE_FIXTURES=$USE_FIXTURES_PROD --build-arg TRELLIS_BASE_URL=$TRELLIS_BASE_URL_PROD --build-arg DEFAULT_PROFILE_SCHEMA_VERSION=$DEFAULT_PROFILE_SCHEMA_VERSION_PROD --build-arg SINOPIA_GROUP=$SINOPIA_GROUP_PROD --build-arg SINOPIA_URI=$SINOPIA_URI_PROD --build-arg AWS_COGNITO_DOMAIN=$AWS_COGNITO_DOMAIN_PROD --build-arg COGNITO_CLIENT_ID=$COGNITO_CLIENT_ID_PROD --build-arg COGNITO_USER_POOL_ID=$COGNITO_USER_POOL_ID_PROD .
```

Comment:
Do we need to add these as new ENV vars in circleci?

```
USE_FIXTURES_DEV
USE_FIXTURES_PROD
USE_FIXTURES_STAGE
```

Reply:
working on it, and yes but if I can get the docker build working locally without these vars, then I will alter circleci config so it just takes defaults for these and also for SINOPIA_GROUP and DEFAULT_PROFILE_SCHEMA_VERSION

Reply:
I ticketed this as #806